test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="php-fpm-client-$RANDOM-$RANDOM"
+docker build -q -t "$client_image" - > /dev/null <<EOF
```

Comment:
I actually just meant a "fat" image that is a locally built version of `debian:jessie` that has already run `apt-get update`. That way different client images wouldn't need to run it. Another option might be to actually publish the client images under the `librarytest` ns as automated builds.